concourse/pipelines/deploy.yml

```diff
@@ -196,7 +238,6 @@ jobs:
     - get: govuk-infrastructure
       passed:
       - deploy-frontend
-      trigger: true
```

Comment:
I hadn't spotted this before - is it intentional?

Reply:
ah it was what Richard suggested but I will re-add that because it did not work for the others.